recovery/README.md

```diff
@@ -167,6 +167,53 @@ spec:
                 name: backup
 ```
 
+## Create Recovery Partition
+
+The recovery partition can be created at install time by defining an extra-manifest MachineConfig, as described here:<br>
+<https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md#deploying-a-site>
+
+The following extra-manifest MachineConfig will create a 50G partition on the specified disk:
+
+```
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-var-recovery-partition
```

Comment:
Should we rather propose a generic name?

Reply:
I was trying to follow MachineConfig conventions. Is there something you'd prefer to see? I don't think the name impacts anything

Reply:
No it does not impact anything. Just wondering what would 98 mean, you can ignore it

Reply:
Hangs here long enough for  people to see :)
/lgtm